northstar/src/seccomp/bpf.rs

```diff
@@ -181,10 +182,29 @@ fn check_platform_requirements() {
     compile_error!("Seccomp is not supported on Big Endian architectures");
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct SockFilter {
```

Comment:
Not sure if this helps much, but `serde` has a `remote` feature to make external types serializable: https://serde.rs/remote-derive.html. With it, I think it would be possible to avoid replacing `sock_filter` with `SockFilter`.

Reply:
Thanks. The remote feature would have worked here. I didn't use it because I wanted to avoid to use type from the c bindings to widespread and prefer the use of an internal type.